src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js

```diff
@@ -26,27 +26,28 @@ function AssemblyChange({ item, creating, handleFieldChange }) {
         collectionSelectorHelpers.getSearchLoading(state.parts)
     );
 
-    const handlePartChange = selectedPart => {
-        handleFieldChange('newPartNumber', selectedPart.partNumber);
+    const handlePartChange = partNumber => {
+        handleFieldChange('newPartNumber', partNumber);
     };
 
     return (
         <>
             {creating ? (
                 <Grid item xs={12}>
-                    <Typeahead
+                    <Search
+                        propertyName="newPartNumber"
                         label="Part"
-                        title="Search for a part"
-                        onSelect={handlePartChange}
-                        items={partsSearchResults}
+                        helperText="use Enter to search"
```

Comment:
do we need need to mention if tab also works

Reply:
tab doesn't do anything in this case.... Do we think it should?